README.md

```diff
@@ -58,6 +58,15 @@ Each Markdown file is associated with a Confluence wiki page with a Markdown com
 
 The above tells the tool to synchronize the Markdown file with the given Confluence page ID. This implies that the Confluence wiki page must exist such that it has an ID. The comment can be placed anywhere in the source file.
 
+### Provide generated-by prompt text in Markdown file
+
+Provide generated-by prompt text in Markdown file via the following optional tag:
+
+```markdown
+<!-- generated-by: Do not edit! Origin can be found <a href="https://git.example.com/project">here</a>. -->
+```
+Alternatively, use the `--generate-by GENERATED_BY` option. The tag takes precedence.
```

Comment:
If you have a short and simple text, this is definitely a feasible approach. I like this much better. There is likely a typo in the parameter name, it should probably read: `generated_by`.

Reply:
> If you have a short and simple text, this is definitely a feasible approach.

I was thinking of also parsing the prompt text but hey, it's intended to be a one-liner hint only.

> I like this much better.

It's already functional if you do not provide any `generated_by` tag. Again, this `generated-by` tag is optional.

And yes, typo.

Reply:
Typo is fixed.